qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/argo-workflows/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "namespace" {
+  description = "deploy argo server on this namespace"
+  type        = string
+  default     = "dev"
+}
+
+variable "argo-workflows-namespace" {
+  description = "deploy argo workflows on this namespace"
+  type        = string
+  default     = "dev"
+}
+
+# variable "node-group" {
```

Comment:
Important that this is enabled. Great work so far!

Reply:
Done :)